rest/route/service.go

```diff
@@ -82,23 +78,23 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/cost/project/{project_id}/tasks").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeTaskCostByProjectRoute(sc))
 	app.AddRoute("/cost/version/{version_id}").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeCostByVersionHandler(sc))
 	app.AddRoute("/distros").Version(2).Get().Wrap(checkUser).RouteHandler(makeDistroRoute(sc))
-	app.AddRoute("/distros/{distro_id}").Version(2).Get().Wrap(superUser, viewDistroSettings).RouteHandler(makeGetDistroByID(sc))
-	app.AddRoute("/distros/{distro_id}").Version(2).Patch().Wrap(superUser, editDistroSettings).RouteHandler(makePatchDistroByID(sc, settings))
-	app.AddRoute("/distros/{distro_id}").Version(2).Delete().Wrap(superUser, removeDistroSettings).RouteHandler(makeDeleteDistroByID(sc))
-	app.AddRoute("/distros/{distro_id}").Version(2).Put().Wrap(superUser, createDistro).RouteHandler(makePutDistro(sc, settings))
-	app.AddRoute("/distros/{distro_id}/setup").Version(2).Get().Wrap(superUser, viewDistroSettings).RouteHandler(makeGetDistroSetup(sc))
-	app.AddRoute("/distros/{distro_id}/setup").Version(2).Patch().Wrap(superUser, editDistroSettings).RouteHandler(makeChangeDistroSetup(sc))
-	app.AddRoute("/distros/{distro_id}/teardown").Version(2).Get().Wrap(superUser, viewDistroSettings).RouteHandler(makeGetDistroTeardown(sc))
-	app.AddRoute("/distros/{distro_id}/teardown").Version(2).Patch().Wrap(superUser, editDistroSettings).RouteHandler(makeChangeDistroTeardown(sc))
-	app.AddRoute("/distros/{distro_id}/execute").Version(2).Post().Wrap(superUser, editDistroSettings).RouteHandler(makeDistroExecute(sc, env))
+	app.AddRoute("/distros/{distro_id}").Version(2).Get().Wrap(editDistroSettings).RouteHandler(makeGetDistroByID(sc))
```

Comment:
Why `viewDistroSettings` to `editDistroSettings`?

Reply:
This is actually superuser > editDistro. View distro is redundant with that